src/main/java/cat/udl/eps/softarch/learnphysics/repository/ExamRepository.java

```diff
@@ -0,0 +1,4 @@
+package cat.udl.eps.softarch.learnphysics.repository;
+
+public interface ExamRepository {
```

Comment:
You should add the annotations and extend the corresponding class in order that this class exposes the http endpoints. It is not doing anything right now.